lib/krane/cluster_resource_discovery.rb

```diff
@@ -101,6 +99,22 @@ def version_for_kind(versions, kind)
       version_override.fetch(kind, latest)
     end
 
+    def get_resource_api_path(api_versions, resource)
+      apiversion = resource['apiversion'].to_s
+      apiversion = 'core/v1' if apiversion == 'v1'
```

Comment:
I think this logic be: `apiversion = "core/#{apiversion} unless apiversion.include?("/")` (though maybe it will never matter since I don't think anything in the core group will ever get bumped)

Reply:
That should work as long as we check for the empty too otherwise that unless would be try prior to 1.20

Reply:
>I don't think anything in the core group will ever get bumped

Is that written down somewhere? I'd be tempted to future-proof here since the additional logic is sensible enough